Modules/Project/Resources/views/index.blade.php

```diff
@@ -87,7 +87,8 @@
                                 <td colspan=4 class="font-weight-bold">
                                     <div class="d-flex justify-content-between">
                                         <div>
-                                            {{ $client->name }}
+                                            {{ $client->name }} 
+                                            {!! ' ('. str_pad($client->client_id , 3, '0', STR_PAD_LEFT) .')'!!}
```

Comment:
```suggestion
                                            {{ ' ('. str_pad($client->client_id , 3, '0', STR_PAD_LEFT) .')' }}
```

@ImRishabhRawat can we write the above line like this? let me know if this works.